packages/next/src/server/config-shared.ts

```diff
@@ -771,6 +775,9 @@ export const defaultConfig: NextConfig = {
     instrumentationHook: false,
     bundlePagesExternals: false,
   },
+  future: {
+    fallbackNodePolyfills: false,
+  },
```

Comment:
If the default value is `false`, then the test below is not doing anything because its also setting it to `false`.